### ISSUE TITLE
mediathekview URL with country protection did not download

### ISSUE THREAD

**Issue by pinkzomo** [ytdl-org/youtube-dl#32020]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x ] I've verified that I'm running youtube-dl version **2021.12.17**
- [ ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ ] I've searched the bugtracker for similar bug reports including closed ones
- [ ] I've read bugs section in FAQ

## Verbose log

```
Updating to version 2021.12.17 ...
Updated youtube-dl. Restart youtube-dl to use the new version.
ith@x1:~/mediathek$ youtube-dl --geo-bypass-country AT https://apasfiis.sf.apa.at/ipad/cms-austria/20170203_1522_in_02_Ludwig-Boltzman_____13908900__o__7430180085__s13970856_Q8C.mp4/playlist.m3u8 --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'--geo-bypass-country', u'AT', u'https://apasfiis.sf.apa.at/ipad/cms-austria/20170203_1522_in_02_Ludwig-Boltzman_____13908900__o__7430180085__s13970856_Q8C.mp4/playlist.m3u8', u'--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 2.7.18 (CPython) - Linux-5.15.0-69-generic-x86_64-with-Ubuntu-22.04-jammy
[debug] exe versions: ffmpeg 4.4.2, ffprobe 4.4.2, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 77.116.176.197 (AT) as X-Forwarded-For.
[generic] playlist: Requesting header
WARNING: Could not send HEAD request to https://apasfiis.sf.apa.at/ipad/cms-austria/20170203_1522_in_02_Ludwig-Boltzman_____13908900__o__7430180085__s13970856_Q8C.mp4/playlist.m3u8: HTTP Error 403: Forbidden
[generic] playlist: Downloading webpage
ERROR: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 634, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 2288, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python2.7/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 467, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 654, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 467, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 654, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)


```

## Description

WRITE DESCRIPTION HERE
Using the URL from mediathekview I tried to download a protected video, i.e. country specific file. The video did not download and output demanded to send bug report.

**Comment by dirkf** [comment]
Generally you shouldn't expect some random manifest to be accessible, especially if access control is involved. Use the actual programme page that is supported by yt-dl's extractor. This looks like the page you wanted and appears to work for me:
```
youtube-dl -v --geo-bypass-country AT 'https://tvthek.orf.at/history/Chemie-und-Physik/13557941/Ludwig-Boltzmann-Physiker-und-Philosoph/13908900'

**Comment by dirkf** [comment]
@pinkzomo ?

**Comment by pinkzomo** [comment]
yes, some pages work but not all.
e.g. 
youtube-dl -v --geo-bypass-country AT https://tvthek.orf.at/history/Technik-und-Digitales/13557940/Der-Verkehr-der-Zukunft/14028515
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'--geo-bypass-country', u'AT', u'https://tvthek.orf.at/history/Technik-und-Digitales/13557940/Der-Verkehr-der-Zukunft/14028515']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 2.7.18 (CPython) - Linux-5.15.0-69-generic-x86_64-with-Ubuntu-22.04-jammy
[debug] exe versions: ffmpeg 4.4.2, ffprobe 4.4.2, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 77.116.239.247 (AT) as X-Forwarded-For.
[orf:tvthek] 14028515: Downloading webpage
[orf:tvthek] 14572360: Downloading m3u8 information
WARNING: Failed to download m3u8 information: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading m3u8 information
WARNING: Failed to download m3u8 information: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading m3u8 information
WARNING: Failed to download m3u8 information: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading m3u8 information
WARNING: Failed to download m3u8 information: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading m3u8 information
WARNING: Failed to download m3u8 information: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading f4m manifest
WARNING: Unable to download f4m manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading f4m manifest
WARNING: Unable to download f4m manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading f4m manifest
WARNING: Unable to download f4m manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading f4m manifest
WARNING: Unable to download f4m manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading f4m manifest
WARNING: Unable to download f4m manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading MPD manifest
WARNING: Failed to download MPD manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading MPD manifest
WARNING: Failed to download MPD manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading MPD manifest
WARNING: Failed to download MPD manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading MPD manifest
WARNING: Failed to download MPD manifest: HTTP Error 403: Forbidden
[orf:tvthek] 14572360: Downloading MPD manifest
WARNING: Failed to download MPD manifest: HTTP Error 403: Forbidden
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/orf.py", line 134, in _real_extract
    self._sort_formats(formats)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 1374, in _sort_formats
    raise ExtractorError('No video formats found')
ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

**Comment by dirkf** [comment]
It seems (from UK) that if you use `--geo-bypass-country AT` you may often be lucky enough to hit a "fake" Austrian IP address that isn't blocked by ORF TVthek. This allows you to get the information about available formats as if you were in Austria, but when you come to access those formats via ORF's CDN system you may hit another level of geo-blocking that isn't bypassed by passing an `X-Forwarded-For: <Austrian IP address>` header, giving 403.

If you don't use the geo-bypass, the extractor can tell from the URLs of the offered formats when you are being given the "Programme only available in Austria" video and raises an error.

But there is weirdness all over. Repeated `--list-formats` for the same show get random 403s, 1 or sometimes 2 or more, when getting the manifest details, affecting different manifests even using the same fake IP. Now I find that I can download your problem show above, whereas last night it failed with 403. Hmmm.

**Comment by dirkf** [comment]
Superseded by #32798.
